fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVRewriteUtil.java

```diff
@@ -62,8 +61,7 @@ public static Collection<Partition> getMTMVCanRewritePartitions(MTMV mtmv, Conne
             return res;
         }
         // check mv is normal
-        if (!(mtmv.getStatus().getState() == MTMVState.NORMAL
-                && mtmv.getStatus().getRefreshState() == MTMVRefreshState.SUCCESS)) {
+        if (mtmv.getStatus().getState() != MTMVState.NORMAL) {
```

Comment:
MTMVState#INIT means in building? May  MTMVState#INIT should not return

Reply:
mtmv.getStatus().getState() == MTMVState.NORMAL && mtmv.getStatus().getRefreshState() !=  MTMVRefreshState.INIT maybe better